grails-app/conf/Config.groovy

```diff
@@ -162,6 +162,10 @@ baselayerServer = [
     wmsVersion: '1.1.1'
 ]
 
+// Set to true if you want to test interaction with new servers. This turns
+// your portal instance into an open proxy and can be dangerous.
+allowAnyHost = false
```

Comment:
Why not put this in the `environments -> development` section and set to true? 
Great work BTW!